src/scrapers/leumi.js

```diff
@@ -119,6 +103,75 @@ async function fetchTransactionsForAccount(page, startDate) {
     }
   }
 
+  return txns;
+}
+
+async function extractPendingTransactionsFromPage(page) {
+  const txns = [];
+
+  const tdsValues = await page.$$eval('#WorkSpaceBox #trTodayActivityNapaTableUpper tr td', (tds) => {
+    return tds.map(td =>
+      ({
+        classList: td.getAttribute('class'),
+        innerText: td.innerText,
+      }));
```

Comment:
@esakal weird syntax, I believe you did that because eslint was shouting at you. 
What it meant was probably:
```node
return tds.map((td) => {
  classList: td.getAttribute('class'),
  innerText: td.innerText,
});
```

Reply:
You suggestion will not work, see image below. It is not related to eslint. You must wrap the resulted object with `(` otherwise it will treat `{` as an opening of a function scope and not an opening of json object notation

![image](https://user-images.githubusercontent.com/4751797/38194896-217659f2-3683-11e8-8a81-3b9072d4e256.png)

Reply:
@esakal my bad, missed the fact you were returning an object.